README.md

```diff
@@ -1,38 +1,38 @@
+# Sudoku Game
 
-# Sudoku Game 
+This repository contains a C++ program for sudoku game.
 
-This repository contains a C++ program for sudoku game. 
-
-# Installation 
+# Installation
 
 1. **Install C++ Compiler:**
-   - Ensure you have a C++ compiler on your machine. For Windows, you can use MinGW or Visual Studio with C++ support. For macOS, you can use Xcode Command Line Tools. for Linux, you can use GCC or Clang. 
+   - Ensure you have a C++ compiler on your machine. For Windows, you can use MinGW or Visual Studio with C++ support. For macOS, you can use Xcode Command Line Tools. for Linux, you can use GCC or Clang.
 2. **Clone the Repository**
-   - Open you temerinal or command prompt [cmd + space]
-   - Clone the repository using the following command: 
+   - Open you terminal or command prompt [cmd + space]
+   - Clone the repository using the following command:
    ```
-      git clone my [GitHub's file for Sudoku] (git@github.com:henrytran07/hard-level-sudoku-board.git) 
+      git clone my [GitHub's file for Sudoku] (git@github.com:henrytran07/hard-level-sudoku-board.git)
    ```
-   - this will download the repository to your local machine 
-## Usage 
+   - this will download the repository to your local machine
+
+## Usage
 
 1. **Compile the Program:**
```

Comment:
Fix the instruction in the readme so the user can replicate your work. Make sure to  test it out yourself by cloning the repo to your local and do the instructions that you typed.